.cirrus.yml

```diff
@@ -42,5 +48,32 @@ release_task:
     - python3.7 -m ensurepip
     - python3.8 -m ensurepip
   build_script: ./make-nix-release.sh
+  script: |
+    #!/usr/bin/env bash
```

Comment:
The shebang should not be necessary. Also, I'm pretty sure a prefix is required here, along the lines of  `upload_script:`